src/main/java/org/broadinstitute/hellbender/utils/python/StreamingPythonScriptExecutor.java

```diff
@@ -330,7 +330,10 @@ public ProcessOutput getAccumulatedOutput() {
     private void initializeTool(final File ackFIFOFile) {
         //  first we need to import the module; no ack expected yet as we haven't initialized the ack fifo
         spController.writeProcessInput(PYTHON_IMPORT_GATK); // no ack generated
-        spController.writeProcessInput(String.format(PYTHON_INITIALIZE_GATK, ackFIFOFile.getAbsolutePath()));
+        spController.writeProcessInput(
+                String.format(PYTHON_INITIALIZE_GATK,
+                        RuntimeUtils.getGATKVersionString(),
```

Comment:
Maybe a constructor argument with the version string should be added to the `StreamingPythonScriptExecutor`; otherwise, downstream toolkits cannot use it.